tools/pythonpkg/tests/extensions/test_httpfs.py

```diff
@@ -1,12 +1,17 @@
 import duckdb
 import os
 import pandas as pd
+from pytest import raises, mark
+
+
+# We only run this test if this env var is set
+pytestmark = mark.skipif(
+    not os.getenv('DUCKDB_PYTHON_TEST_EXTENSION_REQUIRED', False),
+    reason='httpfs extension not available'
```

Comment:
Isn't this already done by `require` ?
I recently learned you can actually see these reasons when you run pytest with the `-rs` flags. (hardly documented anywhere for some reason)

Reply:
Yeah I'm not sure why this environment variable exists, happy to remove this check though